docs/_includes/js/store.js

```diff
@@ -0,0 +1,66 @@
+document.querySelector('.dropdown').onclick = function() {
+
+  if (this.classList.contains('is-shown')) {
+    this.classList.remove('is-shown');
+  } else {
+    this.classList.add('is-shown');
+  }
+};
+
+[].forEach.call( document.querySelectorAll('.dropdown .dropdown-options a'), function(el) {
+  el.addEventListener('click', function() {
+    document.querySelector('.dropdown-selection label').innerHTML = this.innerHTML;
```

Comment:
declare your variables and assign your elements on top of your script and use them where necessary for a cleaner and faster code w/o repetitions.

```js
var dropdownLabel = document.querySelector('.dropdown-selection label');
...
label.innerHTML = this.innerHTML;
```